installer/scripts/omsadmin.sh

```diff
@@ -170,10 +170,11 @@ parse_args()
     local OPTIND opt
 
     for arg in "$@"; do
-        if [ "$arg" = "--collectd" ]; then
-          set -- "$@" "-c"
-        fi
-        shift;
+        shift
+        case "$arg" in
+          "--collectd") set -- "$@" "-c" ;;
+          *)            set -- "$@" "$arg" ;;
+        esac
```

Comment:
I'm not going to give signoff on this, sorry.

First, I'm not 100% sure what `set --` does, nor is Nick. But that's not that important.

What is important is that this script takes a bunch of different parameters, and NONE need to be special cased this way.

I'd like to see one of two things done:

1. Eliminate this code fragment and just accept `-c` for the collectd function. This allows consistency with all of the other parameters, or
2. Refactor the parameter parsing (like the OMS build/configure script) to allow both shorter and longer parameters. In this way, if you REALLY want to support `--collectd`, you can.

Personally, I'd just do the first - I think that's fine. But I'll leave it up to you.